clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -146,6 +146,14 @@ static bool startsNextParameter(const FormatToken &Current,
            Style.BreakInheritanceList != FormatStyle::BILS_BeforeComma));
 }
 
+// Returns \c true if \c Current starts a new operand in a binary operation.
```

Comment:
Did you mean `Current is the right operand of a binary operator`?

Reply:
The wording was meant to match the comment for `startsNextParameter`, but I changed the wording to clarify a bit. 

In a case like this: `op1 * (op2 + op3)`
The function would return true for `(` since that's what starts the next operand for the `*` operation.
It would be a bit inaccurate to say `(` "`is the right operand of a binary operator`"